index.js

```diff
@@ -0,0 +1,73 @@
+const { inspect } = require('util');
```

Comment:
wait if you're using `inspect` do you still need the custom formatting below?

Reply:
just using inspect instead of `JSON.stringify` to help collapse large objects, etc